support/printable.py

```diff
@@ -171,7 +171,7 @@ def main():
     normal1 = compress_normal(normal1)
 
     print("""\
-inline auto is_printable(uint32_t cp) -> bool {\
+FMT_PRINT auto is_printable(uint32_t cp) -> bool {\
```

Comment:
You probably meant `FMT_FUNC`.

Reply:
Yeah lol. I made the same mistake in the real code, but of course that one I actually compiled... Fixed.